ServerTestHost/Program.cs

```diff
@@ -20,6 +20,11 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .ConfigureLogging(logging =>
+                {
+                    logging.ClearProviders();
+                    logging.AddConsole();
+                })
```

Comment:
I guess these are not needed anymore?

Reply:
Probably not.

Reply:
Removed the console logger from the server side project.